### PR TITLE
Add AML search API

### PR DESCRIPTION
**Pull request by mclassic**
Add AML search API to the SDK.

### CODE REVIEW

File: src/Entities/Session.php [hunk 1]
```diff
@@ -180,6 +180,7 @@ public static function allowedResources()
             ResourceType::TYPE_TRANSACTIONS,
             ResourceType::TYPE_TOKENS,
             ResourceType::TYPE_WEBSITES,
+            ResourceType::TYPE_KYC,
```
Comment (adamaltman) [inline src/Entities/Session.php]: I think these are organized in alpha.

**Comment by mclassic** [comment]
php please review

**Review by ashkarpetin — DISMISSED** [review]

File: src/Entities/Session.php (hunk 1 above)
Comment (mclassic) [reply src/Entities/Session.php]: @adamaltman Resolved.

**Comment by mclassic** [comment]
php please review

**Comment by mclassic** [comment]
php please review

File: src/Entities/AmlEntry.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<?php
+
```
Comment (VladimirBerdnik) [inline src/Entities/AmlEntry.php:2]:
```suggestion
/**
 * This source file is proprietary and part of Rebilly.
 *
 * (c) Rebilly SRL
 *     Rebilly Ltd.
 *     Rebilly Inc.
 *
 * @see https://www.rebilly.com
 */
 
```

**Review by VladimirBerdnik — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**